### PR TITLE
Multiselect

### PR DESCRIPTION
**Pull request by APJohns**
Added multiselect component.

**Features:**
- label (visual and hidden)
- items type supports both array of basic types and array of objects
- placeholder
- searchable

Documentation will be in a follow up PR.

### CODE REVIEW

File: projects/epsilon-blueprint/src/lib/multiselect/multiselect.component.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    bindValue="value"
+    bindLabel="label"
+    [placeholder]="placeholder"
+    [searchable]="isSearchable"
+    [clearable]="false"
+    [closeOnSelect]="false"
+    [(ngModel)]="selectedOptions"
+    (change)="emitSelected()"
+  >
+    <!-- <ng-template ng-multi-label-tmp let-items="items" let-clear="clear">
```
Comment (kaseyllabelle) [inline projects/epsilon-blueprint/src/lib/multiselect/multiselect.component.html:19]: Delete or add TODO

File: src/app/modules/component-library/library/multiselect/api/api.component.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<section class="container-fluid">
+  <div class="row">
+    <div class="col">
+
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/multiselect/api/api.component.html:4]: Follow up PR?

File: src/app/modules/component-library/library/multiselect/multiselect.component.html [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<app-component-header
+  headingText="Buttons & Links"
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/multiselect/multiselect.component.html:2]: Update text

File: src/app/modules/component-library/library/multiselect/overview/overview.component.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      <div class="card mb-3">
+        <div class="card-body component-preview">
+          <div class="row">
+            <div class="col-12 col-sm-6 col-md-5 col-lg-3 col-xl-2">
+              <bp-multiselect
+                label="Label Text"
+                [optionItems]="optionItems"
+                bindLabel="label"
+                bindValue="value"
+                placeholder="Select an item"
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/multiselect/overview/overview.component.html:18]: Should we make this plural to reinforce the idea of the multiselect?

File: src/app/modules/component-library/library/multiselect/overview/overview.component.html [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<section class="container-fluid">
+  <div class="row">
+    <div class="col-lg-8 col-xl-7">
+      <p class="component-description">
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/multiselect/overview/overview.component.html:4]: Can we add something here?

File: src/app/modules/component-library/library/multiselect/overview/overview.component.ts [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  styleUrls: ['./overview.component.scss']
+})
+export class OverviewComponent implements OnInit {
+  optionItems = [
+    {
+      label: 'Option 1',
+      value: 'option1'
+    },
+    {
+      label: 'Option 2',
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/multiselect/overview/overview.component.ts:15]: Can we make one of these really long to show how it handles options with lots of text?

File: src/app/modules/component-library/library/multiselect/usage/usage.component.html [hunk 7]
```diff
@@ -0,0 +1,9 @@
+<section class="container-fluid">
+  <div class="row">
+    <div class="col-lg-8 col-xl-7">
+      <p class="component-description">
+
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/multiselect/usage/usage.component.html:5]: Follow up PR?

File: projects/epsilon-blueprint/src/lib/multiselect/multiselect.component.ts [hunk 8]
```diff
@@ -28,13 +29,13 @@ export class MultiselectComponent implements OnInit {
 
   validate() {
     if (!this.label) {
-      let err = new Error('\'label\' is a required Input of bp-multiselect. The label can be visually hidden using the \'isLabelHidden\' property.');
+      let err = new Error('\'label\' is a required Input of bp-multiselect. The label can be visually hidden using the \'isLabelHidden property.');
```
Comment (kaseyllabelle) [inline projects/epsilon-blueprint/src/lib/multiselect/multiselect.component.ts:32]: Was this intentional?

File: src/app/modules/component-library/library/multiselect/api/api.component.html (hunk 2 above)
Comment (APJohns) [reply src/app/modules/component-library/library/multiselect/api/api.component.html:4]: Documentation will be in a follow up PR (See PR description)

File: src/app/modules/component-library/library/multiselect/overview/overview.component.html (hunk 5 above)
Comment (APJohns) [reply src/app/modules/component-library/library/multiselect/overview/overview.component.html:4]: Documentation will be in a follow up PR (See PR description)

File: src/app/modules/component-library/library/multiselect/usage/usage.component.html (hunk 7 above)
Comment (APJohns) [reply src/app/modules/component-library/library/multiselect/usage/usage.component.html:5]: Documentation will be in a follow up PR (See PR description)

**Pull request merged**